packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/configuration/onPremise/onPremisePcc/dedicatedCloud-datacenter-drp-onPremise-onPremisePccStep.controller.js

```diff
@@ -1,34 +1,7 @@
-import get from 'lodash/get';
-
-import {
-  DEDICATEDCLOUD_DATACENTER_DRP_IP_BLOCK_REG_EXP,
-  DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS,
-  DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS,
-  DEDICATEDCLOUD_DATACENTER_ZERTO,
-} from '../../../dedicatedCloud-datacenter-drp.constants';
-
 export default class {
   /* @ngInject */
-  constructor(
-    $state,
-    $timeout,
-    $translate,
-    $window,
-    dedicatedCloudDrp,
-    ovhUserPref,
-    DEDICATED_CLOUD_CONSTANTS,
-  ) {
+  constructor($state) {
```

Comment:
Couldn't by added to the routing file?

Here is what I have in mind:

```js
resolve: {
  drpInformations: /* @ngInject */ ($transition$) =>
    $transition$.params().drpInformations,
}
```

It can prevent you to inject the `$state` inside your controller.